go.work

```diff
@@ -0,0 +1,9 @@
+go 1.23.6
```

Comment:
Could be updated to `1.24.0`. See https://github.com/OpenSlides/openslides-autoupdate-service/pull/1147

Reply:
Does the versioning here mean that all repos in the `use` block have to use the same version and vice versa?